### PR TITLE
add 7 locales to CPG v2

### PR DESCRIPTION
**Pull request by peiying2**
In addition to en-US, de, es, fr, hi, pt-BR, and zh-TW are added. 

NOTE:
- there is no "es" folder (I thought we had), so right now the file lives in es-ES
- also this file lives under a multiple folder levels and there is nothing else but this file. I hope I processed it correctly.

### CODE REVIEW

**Comment by flodolo** [comment]
Honestly, I'm not surprised they broke those strings, especially if they didn't get any instructions on how placeholders were going to be replaced. On other hand, I'll be surprised if any of the translations is good… 
https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html#L403-L418

File: fr/mozorg/about/governance/policies/participation.lang [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+;Hurtful or harmful language related to:
+Des termes blessants ou offensants liés à :
+
+
+;Other attributes
+d’autres attributs ;
+
+
+;is not acceptable. This includes deliberately referring to someone by a gender that they do not identify with, and/or questioning the legitimacy of an individual’s gender identity. If you’re unsure if a word is derogatory, don’t use it. This also includes repeated subtle and/or indirect discrimination; when asked to stop, stop the behavior in question.
+n’est pas acceptable. Cela comprend le fait de désigner quelqu’un par un genre auquel cette personne ne s’identifie pas ou de questionner la légitimité de son identité de genre. Si vous n’êtes pas sûr(e) de savoir si un terme est péjoratif ou pas, ne l’utilisez pas. Cela comprend également les discriminations répétées subtiles ou indirectes.; when asked to stop, stop the behavior in question.
```
Comment (TheoChevalier) [inline fr/mozorg/about/governance/policies/participation.lang]: `; when asked to stop, stop the behavior in question.` has to be removed

**Comment by TheoChevalier** [comment]
> I don't speak French, but I quickly checked and I don't think the team will be happy, at least looking at punctuation.

Yeah, all the nbsp chars have been replaced with regular spaces somehow + a few style mistakes (they are all listed in our styleguide):

- événement vs. évènement
- site Web vs. site web
- we’re translating “e-mail”
(Haven’t done a full review, just had a quick look)

**Comment by peiying2** [comment]
@flodolo Should I just insert replace & with &amp;? It appears that several locales are replaced with & even though only hi-IN reports it as an error.

Good point on hi-HI on Pootle. Will send Leandro an email on this.

**Comment by flodolo** [comment]
The problem is that they touched the reference string, don't really care about the translation (even if it should probably remain the same).

**Comment by flodolo** [comment]
@TheoChevalier There is other stuff to fix, so we'll land this as original, at least we'll be able to see the diff.

**Pull request merged**